tests/unit/routes/test_credential.py

```diff
@@ -132,6 +134,46 @@ def test_update_200(self, create_success_response, test_api_client_persistent):
         assert decrypt_value(previous_values[0]["password"]) == "pass"
         assert previous_values[1] is None
 
+    def test_update_group_ids_200(self, create_success_response, test_api_client_persistent):
+        """
+        Test credential update (success)
+        """
+        test_api_client, _ = test_api_client_persistent
+        response_dict = create_success_response.json()
+        credential_id = response_dict["_id"]
+        updated_group_ids = ["63eda344d0313fb925f7883b", "63eda344d0313fb925f7883a"]
```

Comment:
Are you planning to override featurebyte-app credential service to validate the group ID provided by the caller?

Reply:
would it be possible to just add a field validator to CredentialRead? 
Check that it is None or that it is a list of objectId.

Reply:
I think current payload schema already validate the object. What is missing here is the validate the group object exists or not in the mongo.

Reply:
Will do